src/lib/keychain/insecure.js

```diff
@@ -12,15 +12,27 @@ import path from 'path';
  */
 import type { Keychain } from './keychain';
 
-export default class Secure implements Keychain {
+export default class Insecure implements Keychain {
 	file: string;
+	passwords: Object;
 
 	constructor( file: string ) {
 		// only current user has read-write access
-		const rw = 0o600;
+		let rw;
+		switch ( process.platform ) {
```

Comment:
Would be good to have an inline comment referencing why we need this special handling.

Reply:
Fortunately found a better way to use the built in constants so this special handling isn't necessary.